_data/simple-icons.json

```
@@ -5761,6 +5761,12 @@
             "source": "https://www.hashicorp.com",
             "guidelines": "https://www.hashicorp.com/brand"
         },
+        {
+            "title": "HashiCorp Nomad",
+            "hex": "60DEA9",
+            "source": "https://www.hashicorp.com/products/nomad",
+            "guidelines": "https://www.hashicorp.com/brand"
```

```suggestion
            "title": "Nomad",
            "hex": "00CA8E",
            "source": "https://www.hashicorp.com/brand",
            "guidelines": "https://www.hashicorp.com/brand"
```

This'll need to be re-ordered alphabetically in order to keep the linter happy.

---

Thanks for the feedback, working on it.